python/nx-cugraph/nx_cugraph/algorithms/components/connected.py

```diff
@@ -75,7 +84,7 @@ def connected_components(G):
     groups = _groupby(labels, node_ids)
     it = (G._nodearray_to_set(connected_ids) for connected_ids in groups.values())
     # TODO: PLC doesn't handle isolated vertices yet, so this is a temporary fix
```

Comment:
Sorry for not catching this earlier, but PLC added support for isolated vertices in 23.12.  I updated nx-cugraph to use the new feature in [this PR](https://github.com/rapidsai/cugraph/pull/4077).

Reply:
Cool, thanks. Once that PR is merged, should we see if we can remove this (and similar) "temporary fix" code?